### ISSUE TITLE
Ukrainian translation

### ISSUE THREAD

**Issue by troyane** [dwmkerr/hacker-laws#236]
Great reading! Would like to help to make Ukrainian translation :ukraine:

**Comment by dwmkerr** [comment]
Awesome! @troyane if you sign in to GitLocalize and let me know when you've got an account I'll make you the author of the Ukrainian translation, thanks so much!

**Comment by troyane** [comment]
@dwmkerr never used GitLocalize before. But I've already signed in, https://gitlocalize.com/users/troyane

**Comment by dwmkerr** [comment]
Excellent, I've just added you as a moderator!

![image](https://user-images.githubusercontent.com/1926984/75091920-835f8580-55ad-11ea-8b75-e3ff27918bfb.png)

I'll also link you to the convo we are having on translations, #234

**Comment by troyane** [comment]
@dwmkerr my friend, how could I add translators/moderators for language?

**Comment by dwmkerr** [comment]
@troyane just let me know if you need to add anyone else or raise a new issue if needs be! Closing this one for now

**Comment by troyane** [comment]
@dwmkerr What does it means that you are going to close it? Apply changes and publish ukrainian translation OR do nothing and close issue? It is still in progress. Have only few hours per week for this, and found few of the rules a bit difficult to translate. Would like to add translator(s) for more help.

**Comment by dwmkerr** [comment]
Hi @troyane only that I'm closing this issue - feel free to let me know when you are ready with any translations - partial is fine:

https://github.com/dwmkerr/hacker-laws#translations

Lots of these are partial anyway :) If you need more people adding to GitLocalise or anything like that just open a new issue with their names etc and I can add them. You can either open a PR to add the translation link or if you prefer I can do it!

**Comment by dwmkerr** [comment]
(Re-opening until there's a link on the homepage)

**Comment by troyane** [comment]
@dwmkerr could you please add @HelgaLastivka to Ukrainian translators, please?

**Comment by dwmkerr** [comment]
Hi @troyane @helgalastivka can Lastiva please login to GitLocalise so that their account is created and then I can add to the translators? Thanks!

**Comment by HelgaLastivka** [comment]
Hi @dwmkerr, all is done.

**Comment by dwmkerr** [comment]
Nice - I *think* @HelgaLastivka you should be able to edit any translation now:

![image](https://user-images.githubusercontent.com/1926984/96847894-4d4d0b00-1486-11eb-902d-97484793201e.png)

if you have any issues let me know - in the meantime I'll add a Ukrainian link to the front-page etc!

**Comment by dwmkerr** [comment]
Add @troyane @HelgaLastivka I've made a PR to add to the front page #320 

Would you mind letting me know if I've got the Ukrainian word for the Ukrainian language correct (I had to use Wikipedia), also if you prefer only first names/handles, etc, just give me a shout (I've used the names as shown in your profiles)!

**Comment by troyane** [comment]
@dwmkerr Looks good!

**Comment by troyane** [comment]
@dwmkerr Can you help to figure out how to resolve merge conflict in GitLocalize? 

It shows this badge
![image](https://user-images.githubusercontent.com/477468/115737477-d6d98900-a394-11eb-8611-2a880606658c.png)

And somehow can't do anything.

Click on red-highlighted text:
![image](https://user-images.githubusercontent.com/477468/115737867-36d02f80-a395-11eb-8771-d85c51154af3.png)

Choose "Link to segment", link it:
![image](https://user-images.githubusercontent.com/477468/115737987-4c455980-a395-11eb-9c65-ee1d2d982b36.png)

and all remains the same:
![image](https://user-images.githubusercontent.com/477468/115738067-5ebf9300-a395-11eb-9ac2-d81739c97e42.png)

And there seems to be no other controls on a page available.

Any clue?

**Comment by dwmkerr** [comment]
If you click on the text in red do you get these options?

![image](https://user-images.githubusercontent.com/1926984/115810439-7e4fcd80-a420-11eb-8976-10a5af6ded6b.png)

I think you have to click on things to see the 'link' or 'edit' options

**Comment by troyane** [comment]
@dwmkerr yeah, I've tried. Can't understand what is wrong. Or maybe there is something so much unintuitive that I'm missing.
![ezgif com-gif-maker (2)](https://user-images.githubusercontent.com/477468/115897588-40c16380-a465-11eb-8185-e0086b7a6223.gif)

**Comment by dwmkerr** [comment]
I've pinged the Git Localise team to see if they can chip in on this issue and how to fix it!

**Comment by ilyaspiridonov** [comment]
Looking into it
@svasilenkov do you think a rest would fix this?

**Comment by dwmkerr** [comment]
@troyane I think that it is working now by the way! Thanks @ilyaspiridonov!
